docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -649,7 +653,7 @@ class ComponentExample extends React.PureComponent<IComponentExampleProps, IComp
               <div style={{ flex: '0 0 auto' }}>
                 <ComponentControls
                   anchorName={this.anchorName}
-                  examplePath={examplePath}
+                  examplePath={currentExamplePath || examplePath}
```

Comment:
just for the sake of eliminating this a bit confusing expression (in a sense that it is not immediately evident what is the difference between these two), we could use 

```jsx
<ComponentControls
  ...
  examplePath = {this.sourceCodeMgr.currentPath}
  ... />
```
and eliminate `currentExamplePath` from all places - as `this.sourceCodeMgr` already serves as state manager, and state updates are called on path change